pkg/interpreter/builtin/misc.go

```diff
@@ -191,3 +194,56 @@ func Prog(i *types.Interpreter, args []interface{}) (interface{}, error) {
 	// return result of last statement in prog
 	return res, nil
 }
+
+func ToString(arg interface{}) string {
+	switch arg.(type) {
```

Comment:
```suggestion
	switch v := arg.(type) {
```

тогда сможем не писать `arg.(ast.Atom)` и `arg.(ast.List)`

Reply:
Какая-то магия)